### DISCUSSION TITLE
写了个小玩意，可以在 OpenWrt 下自动把广告规则合并至 hosts

### DISCUSSION THREAD

**Discussion by Aethersailor** [TG-Twilight/AWAvenue-Ads-Rule#67] [Show and tell]
https://github.com/Aethersailor/OpenWrt-AutoUpdateHosts

这样就不用套娃其他插件了，直接用 dnsmasq 就可以去广告

**Comment by TG-Twilight** [comment]
感谢来信，我认真查看了这个仓库，我认为这是一个非常棒的项目！

我会在本仓库的“推荐的工具”中向人们推荐它！同时亦会在我们的官方网站推荐！

请问你有Telegram账户吗？我们希望你可以加入我们的 [群组](https://t.me/AWAvenueAdsChat) 进行更多的交流，谢谢。

祝你生活愉快！

**Reply by Aethersailor** [reply]
感谢您的认可！ : )
我一直在你们的Telegram群组中。

**Reply by TG-Twilight** [reply]
but we don't know yours username.  : /

**Comment by TG-Twilight** [comment]
WHAT CAN I SAY？

怎么归档了？

**Comment by Aethersailor** [comment]
> WHAT CAN I SAY？
> 
> 
> 
> 怎么归档了？

因为这个工具实在是脱裤子放屁，把拉取指令添加进计划任务就能有一样的效果。。
